src/components/AppBarComponents/PageList.js

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import Typography from "@mui/material/Typography";
+import { Grid } from "@mui/material";
+import pages from "./pages";
+
+const PageList = () => {
+  return pages.map((page) => {
+    return (
```

Comment:
you dont need to use return inside map if there is one object inside .map(()=>{<Grid>....</Grid>}) should works.